src/test/resources/features/13-condition/conditions.feature

```diff
@@ -18,31 +18,31 @@ Feature: Patient's Conditions Managment
       | active   |
       | inactive |
 
-  @ignore
+  @run
   Scenario: Set first condition to inactive
     And User clicks on set inactive button
     Then System should move condition to inactive section
     And Quit browser
 
-  @ignore
+  @run
   Scenario: Set first condition to active
     And User clicks on set active button
     Then System should move condition to active section
     And Quit browser
 
-  @ignore
+  @run
```

Comment:
these can only be enabled in the same change that introduces their respective step descriptions